pkg/util/policy/agent/agent.py

```diff
@@ -284,138 +258,231 @@ def make_new_full_ruleset(current_rules, new_rules):
 
     return rules
 
+
 def make_rules(addr_scheme, policy_def, policy_id):
     """
     Return dictionary with rules that should be pre-pended to given chains.
 
     The chain names are the keys to the dict, the values are lists of rules.
```

Comment:
This line only refers to one of the function argument. This was fine in the past when it only had a single argument. Please add a note in there saying that this refers to 'policy_def'.